src/main/java/hellfirepvp/astralsorcery/common/constellation/BaseConstellation.java

```diff
@@ -40,6 +44,18 @@ public StarLocation addStar(int x, int y) {
         return null;
     }
 
+    @Override
+    public IConstellation setTexture(AbstractRenderableTexture tex) {
```

Comment:
Both 'AbstractRenderableTexture' and 'RenderType' are client-only classes, this will crash a dedicated server.
Consider making a dedicated class for the background texture + type

Reply:
Little stuck on this. How could I make a dedicated class without running into the same issue of trying to use client-only classes on server code?

Is this something that could be fixed by @OnlyIn annotations?